roles/lokole/defaults/main.yml

```diff
@@ -1,5 +1,7 @@
 # Info needed to install Lokole
-lokole_version: "0.1.24"
+lokole_version: "0.1.26"
+lokole_admin_user: admin
```

Comment:
@c-w @aidan-fitz

IIAB's universal recommendation for OOB login to apps is: Admin/changeme

(Can we specify the username start with an uppercase `A` above, to be sure?)

Reply:
I made this change to branch `add-lokole-admin-user`.